main.go

```diff
@@ -535,7 +532,7 @@ func updateNewSummaryTypeScreenshotName(testItem map[string]interface{}, testLog
 			log.Printf("Screenshot renamed: %s => %s", filepath.Base(origScreenshotPth), filepath.Base(newScreenshotPth))
 		}
 	}
-	return origScreenshotPth, screenshotExists, nil
+	return origScreenshotPth, true, nil
```

Comment:
i think there is no need for the boolean return value.
if there is an error we populate the err variable, if the screenshot is found we populate the origScreenshotPth, otherwise we can leave it empty